docs/maui_support.md

```diff
@@ -0,0 +1,46 @@
+# Migrating from Xamarin.Forms to .NET MAUI
+
+upgrade-assistant include preview support for Xamarin.Forms to .NET MAUI migration. Please note this tool only has preview support and more features coming soon. This tool helps with the migration progress but you will still require to make manual changes to the solution to make it run successfully with .NET MAUI.
```

Comment:
"We are currently building out support for upgrade-assistant to handle Xamarin.Forms to .NET MAUI migrations. Please note...."?